app/views/partials/covid/covidMessage.scala.html

```diff
@@ -0,0 +1,27 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@()(implicit messages: Messages)
+<section class="warning-banner nudge">
+    <h3 class="heading-medium">@messages("covid.line1")</h3>
+    <p>@messages("covid.line2")</p>
+    <div class="list list-bullet">
```

Comment:
could you change this tag to be 'ul' instead - I think there was an accessibility reason around it